docs/sdk/access-control/encryption.md

```diff
@@ -73,9 +73,9 @@ Steps to Encrypt
 
 1. Obtain an `authSig` and create an access control condition.
 2. Encrypt the static content (string, file, zip, etc...) using `LitJsSdk.encryptString` to get the `ciphertext` and `dataToEncryptHash`.
-3. Finally, store the `ciphertext`, `dataToEncryptHash` and other metadata: `accessControlConditions` (or other conditions eg: `evmContractConditions`) and `chain`. You'll need to pass in these
+3. Finally, store the `ciphertext`, `dataToEncryptHash` and other metadata: `accessControlConditions` (or other conditions eg: `evmContractConditions`) and `chain`.
 
-#### Access Control & AuthSig
+#### Passing Access Control Conditions
 
 In this example, we will set the accessControlConditions on if a wallet has at least 0.000001 ETH:
```

Comment:
Minor comment on sentence structure here:

'In this example, our access control condition will check if a wallet has at least 0.000001 ETH'